docs/source/host/server-setup/linux-server.md

```diff
@@ -9,8 +9,8 @@ Packages are currently available for the following distributions:
 ### Ubuntu Server 18.04 LTS
 
 ```sh
-wget https://cdn.vircadia.com/dist/domain-server/ubuntu/athena-server_0.86.0-k2-20200328-13f3ef2-0ubuntu1-1_amd64.deb
-sudo apt-get update && sudo apt-get install ./athena-server_0.86.0-k2-20200328-13f3ef2-0ubuntu1-1_amd64.deb
+wget https://cdn.vircadia.com/dist/domain-server/ubuntu/vircadia-server_2020.2.0-asteria-20200606-db9139f-0ubuntu1-1_amd64.deb
+sudo apt-get update && sudo apt-get install ./vircadia-server_2020.2.0-asteria-20200606-db9139f-0ubuntu1-1_amd64.deb
```

Comment:
Check hashes: I can't see db9139f in the repo.

Reply:
This is from Heather's repo. It was generated on hers.

Reply:
Once we go gold, I can generate another, but for now that server should work just fine.